src/main/java/org/jenkinsci/plugins/configfiles/properties/AbstractPropertiesProvider.java

```diff
@@ -0,0 +1,53 @@
+package org.jenkinsci.plugins.configfiles.properties;
+
+import com.cloudbees.plugins.credentials.common.StandardUsernameCredentials;
+import hudson.FilePath;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import org.jenkinsci.lib.configprovider.AbstractConfigProviderImpl;
+import org.jenkinsci.lib.configprovider.model.Config;
+import org.jenkinsci.lib.configprovider.model.ContentType;
+import org.jenkinsci.plugins.configfiles.properties.security.CredentialsHelper;
+import org.jenkinsci.plugins.configfiles.properties.security.HasPropertyCredentialMappings;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+
+public abstract class AbstractPropertiesProvider extends AbstractConfigProviderImpl {
+
+    @Override
+    public ContentType getContentType() {
+        return ContentType.DefinedType.PROPERTIES;
+    }
+
+    protected String loadTemplateContent() {
+        InputStream in = null;
+        try {
+            in = AbstractPropertiesProvider.class.getResourceAsStream("settings-tpl.xml");
+            return org.apache.commons.io.IOUtils.toString(in, "UTF-8");
```

Comment:
what is that? the .tpl is supposed to be here or not? should be part of the source code?

Reply:
My bad. This part has been removed, default properties settings are now empty.